packages/editor-ui/src/views/WorkflowHistory.vue

```diff
@@ -22,6 +21,12 @@ type WorkflowHistoryActionRecord = {
 	[K in Uppercase<WorkflowHistoryActionTypes[number]>]: Lowercase<K>;
 };
 
+const enum WorkflowHistoryVersionRestoreModalActions {
+	restore = 'restore',
+	deactivateAndRestore = 'deactivateAndRestore',
+	cancel = 'cancel',
+}
```

Comment:
We should avoid raw enums.

- https://fettblog.eu/tidy-typescript-avoid-enums/
- https://www.executeprogram.com/blog/typescript-features-to-avoid
- https://dev.to/ylerjen/use-typescript-const-assertion-instead-of-enums-mfn

Reply:
Yes, I'm aware and also eslint tells you about it that's why it is `const enum` and not just `enum`